app/controllers/api/v1/list_restaurants_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::ListRestaurantsController < ApplicationController
+  def index
+    restaurants = Restaurant.all
+    render json: {entries: restaurants}
```

Comment:
Alright so it´s ok to have RestaurantsController even if we have another one in the regular controllers?